doc/user/pathd.rst

```diff
@@ -534,7 +534,7 @@ retrieved via PCEP a random number based name is generated.
    Display PCC information.
 
 
-.. clicmd:: show sr-te pcep session [NAME]
+.. clicmd:: show sr-te pcep session [json] [NAME]
```

Comment:
Reading this, it seems like the session name should be put after json, e.g.: `show sr-te pcep session json test`.

Reply:
the doc was wrong, but the implem was ok. sorry. fixing it.